src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -76,25 +76,7 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
                 commit({type: ModelCalibrateMutation.CalibrateResultFetched, payload: data});
                 commit({type: ModelCalibrateMutation.WarningsFetched, payload: data.warnings});
 
-                if (data && data.plottingMetadata && data.plottingMetadata.barchart.defaults) {
-                    const defaults = data.plottingMetadata.barchart.defaults;
-                    const unfrozenDefaultOptions = Object.keys(defaults.selected_filter_options)
-                        .reduce((dict, key) => {
-                            dict[key] = [...defaults.selected_filter_options[key]];
-                            return dict;
-                        }, {} as Dict<FilterOption[]>);
-
-                    commit({
-                            type: "plottingSelections/updateBarchartSelections",
-                            payload: {
-                                indicatorId: defaults.indicator_id,
-                                xAxisId: defaults.x_axis_id,
-                                disaggregateById: defaults.disaggregate_by_id,
-                                selectedFilterOptions: unfrozenDefaultOptions
-                            }
-                        },
-                        {root: true});
-                }
+                selectFilterDefaults(data, commit, "updateBarchartSelections")
```

Comment:
```suggestion
                selectFilterDefaults(data, commit, PlottingSelectionsMutations.updateBarchartSelections)
```

Reply:
To do this I would have to change `type PlottingSelectionsMutation = Mutation<PlottingSelectionsState>` to an enum

Reply:
Ah, sorry I assumed it already was an enum, e.g. like ProjectMutations etc. Most of the other mutations types follow the enum pattern, so I think it would be good to make that change unless there's some good reason why we have it the other way in this case.